packages/type-definitions/src/index.ts

```diff
@@ -119,9 +122,13 @@ const defaultTypesBundle: OverrideVersionedType[] = [
     types: types10800,
   },
   {
-    minmax: [10900, undefined],
+    minmax: [10900, 11199],
     types: types10900,
   },
+  {
+    minmax: [11200, undefined],
+    types: types11200,
+  },
```

Comment:
do we know for sure this is going to be the runtime version with which this is released?

Reply:
No, we don't. The current release of KILT won't have the provider pallet deployed. The provider template runtime does. Maybe I should move these definitions to the new repo, and add it here only when it's going to be included? Otherwise it's really difficult to test this.

Reply:
Mhh yeah maybe this is the way then.

Reply:
Metadata V15 is not out yet. V14 only exposes stuff that is directly used by the runtime. The only way to make it work would be to expose a config pallet which exposes to metadata whatever stuff it is passed to its `Config` trait. Not sure what the downsides of that would be. For one definitely WASM size.